### PR TITLE
Updates documentation

### PR DESCRIPTION
**Pull request by daarende**
This PR contains the changes made to the documentation after the last remarks of KU Leuven

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -11,6 +11,20 @@ Anton Van Rompaey en Gert Verstraeten.
 The code has been further adjusted by Marijn Van Den Broeck and Jeroen Degerickx
 and several students.
 
+Since 2017, updates to the Free Pascal code and the maintenance of
+the Github WaTEM/SEDEM site is now mostly done at `Fluves <www.fluves.com>`_,
```
Comment (Sachagobeyn) [inline AUTHORS.rst:15]: is now mostly done **by**

File: AUTHORS.rst [hunk 2]
```diff
@@ -11,6 +11,20 @@ Anton Van Rompaey en Gert Verstraeten.
...
+funded by two organisations of the
+Flemish Government:
+
+- `Departement Omgeving <https://omgeving.vlaanderen.be>`_
+- `Vlaamse Milieu Maatschappij (VMM) <https://www.vmm.be/>`_
+
+The contributors of Fluves, Departement Omgeving and VMM are Johan Van de Wauw, Daan Renders,
+Sacha Gobeyn, Niels De Vleesschouwer, Stijn Van Hoey and Jarne Amerlinck, Dries Luts,
```
Comment (Sachagobeyn) [inline AUTHORS.rst:23]: Niels De Vleeschouwer

File: CONTACT.rst [hunk 3]
```diff
@@ -2,8 +2,13 @@
...
-cn-ws@omgeving.vlaanderen.be
+For technical questions, we refer to the documentation and to the frequently asked
+questions in the documentation. If you have a technical issue with running the
+model, or if you encounter a bug, please use the issue-tracker on github:
+https://github.com/watem-sedem/watem-sedem/issues.
 
-For technical questions, we refer to the frequently asked questions in the
-documentation or to the issue-tracker on git.
\ No newline at end of file
+If you have questions about the history or concept of the model that are not answered in
```
Comment (Sachagobeyn) [inline CONTACT.rst:10]: this should have a reference no? or is this being discussed with Gert, if not: ask him how to contact them

File: docs/formulas-units.rst [hunk 4]
```diff
@@ -12,11 +12,11 @@ between the units used in the :ref:`model description<modeldescription>`,
 outputs<modeloutput>`. This overview can help interpret the outputs of WaTEM/SEDEM.
 
 As explained in :ref:`the model description <WS>` of WaTEM/SEDEM, the amount of erosion and deposition in
-each pixel is determined by comparing the mean annual soil erosion :math:`E`
-and incoming sediment :math:`S_i` with the transport capacity :math:`TC`.
+each pixel, as well as the amount of sediment leaving a pixel downstream (So) is determined by comparing
```
Comment (Sachagobeyn) [inline docs/formulas-units.rst:15]: So, :math:`S_o`

File: docs/formulas-units.rst [hunk 5]
```diff
@@ -72,6 +72,12 @@ The model outputs `incoming sediment` :math:`S_i` (:ref:`SediIn
 Note that :math:`S_e` is equal to :math:`S_o`, yet masked respectively for
 river and land pixels.
 
+WaTEM/SEDEM outputs include an erosion map that expresses E in kg/m²/year,
```
Comment (Sachagobeyn) [inline docs/formulas-units.rst:75]: :math:`E``

Comment (Sachagobeyn) [reply docs/formulas-units.rst:75]: does "m²" render in sphinx?

**Review by Sachagobeyn — APPROVED** [review]
check comments please before merging

File: CONTACT.rst (hunk 3 above)
Comment (daarende) [reply CONTACT.rst:10]: yes this contact point shall be added when it is created by Gert. For now we leave it like this.

File: AUTHORS.rst (hunk 1 above)
Comment (daarende) [reply AUTHORS.rst:15]: adjusted

File: docs/formulas-units.rst (hunk 5 above)
Comment (daarende) [reply docs/formulas-units.rst:75]: adjusted

**Pull request merged**